examples/finetune/finetune.cpp

```diff
@@ -1288,6 +1288,7 @@ static void train_print_usage(int argc, char ** argv, const struct train_params
     fprintf(stderr, "  --model-base FNAME         model path from which to load base model (default '%s')\n", params->fn_model_base);
     fprintf(stderr, "  --lora-out FNAME           path to save llama lora (default '%s')\n", params->fn_lora_out);
     fprintf(stderr, "  --only-write-lora          only save llama lora, don't do any training.  use this if you only want to convert a checkpoint to a lora adapter.\n");
+    fprintf(stderr, "  --n-gpu-layers N           Number of model layers to offload to GPU (default 0).\n");
```

Comment:
This was put on the https://github.com/ggerganov/llama.cpp/blob/master/common/train.cpp